zerver/management/commands/export_search.py

```diff
@@ -99,8 +114,21 @@ def handle(self, *args: Any, **options: Any) -> None:
             limits &= Q(date_sent__gt=options["after"])
         if options["before"]:
             limits &= Q(date_sent__lt=options["before"])
-        if options["sender"]:
-            print(options["sender"])
+        if options["recipient"]:
+            user_profiles = [get_user_by_delivery_email(e, realm) for e in options["recipient"]]
+            for user in user_profiles:
+                # Users need to not be long-term idle for the
+                # UserMessages to be a judge of which messages they
+                # received.
+                reactivate_user_if_soft_deactivated(user)
+            limits &= reduce(
+                or_,
+                [
+                    Q(usermessage__user_profile_id=get_user_by_delivery_email(e, realm).id)
+                    for e in options["recipient"]
+                ],
+            )
```

Comment:
`usermessage__user_profile_id__in=[user_profile.id for user_profile in user_profiles]` might be a simpler query for the small sets of users that we expect to see often here?

Reply:
I'm certain those have the exact same query plan, but sure.

Reply:
Yeah, I'm not entirely sure what postgres will do, but in any case it felt easier to read.